pom.xml

```
@@ -412,7 +365,7 @@
         </snapshotRepository>
     </distributionManagement>
     <properties>
-        <ballerina.version>0.991.0</ballerina.version>
+        <ballerina.version>1.0.0-rc1-SNAPSHOT</ballerina.version>
```

```suggestion
        <ballerina.version>1.0.0</ballerina.version>
```
Should support GA version of Ballerina.